### ISSUE TITLE
Curl stats visible when doing nerdctl pull on openSUSE

### ISSUE THREAD

**Issue by thehejik** [rancher-sandbox/rancher-desktop#2544]
### Actual Behavior

Curl stats are visible on stdout when doing `nerdctl pull busybox` on openSUSE distros. It behaves correctly in `rdctl shell` env and also on Ubuntu 20.04 (gnome).

Rancher-desktop version `1.4.1-341-gafb05071` built from `main` branch installed on openSUSE 15.3 (gnome) or Tumbleweed (kde) over RPM.

### Steps to Reproduce

- install rd main from RPM and init with default values.
- in terminal run `nerdctl pull busybox`

### Result

Looks like these stats came from cuRL:
```bash
$ nerdctl pull busybox
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100    73    0    58  100    15   2520    651 --:--:-- --:--:-- --:--:--  3650
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100    61    0    52  100     9   1668    288 --:--:-- --:--:-- --:--:--  2033
docker.io/library/busybox:latest:                                                 resolved       |++++++++++++++++++++++++++++++++++++++| 
index-sha256:3614ca5eacf0a3a1bcc361c939202a974b4902b9334ff36eb29ffe9011aaad83:    exists         |++++++++++++++++++++++++++++++++++++++| 
manifest-sha256:dcdf379c574e1773d703f0c0d56d67594e7a91d6b84d11ff46799f60fb081c52: exists         |++++++++++++++++++++++++++++++++++++++| 
config-sha256:62aedd01bd8520c43d06b09f7a0f67ba9720bdc04631a8242c65ea995f3ecac8:   exists         |++++++++++++++++++++++++++++++++++++++| 
elapsed: 1.3 s                                                                    total:   0.0 B (0.0 B/s)                            1.4.1-341-gafb05071
```

### Expected Behavior

Stats are probably not relevant and shouldn't be visible in `nerdctl pull` output.

### Additional Information
* Works ok on openSUSE with stable rd 1.3.0
* Works ok on ubuntu 20.04 (gnome)
* Works correctly in `rdctl shell` env:
```bash
$ nerdctl pull busybox
docker.io/library/busybox:latest:                                                 resolved       |++++++++++++++++++++++++++++++++++++++| 
index-sha256:3614ca5eacf0a3a1bcc361c939202a974b4902b9334ff36eb29ffe9011aaad83:    exists         |++++++++++++++++++++++++++++++++++++++| 
manifest-sha256:dcdf379c574e1773d703f0c0d56d67594e7a91d6b84d11ff46799f60fb081c52: done           |++++++++++++++++++++++++++++++++++++++| 
config-sha256:62aedd01bd8520c43d06b09f7a0f67ba9720bdc04631a8242c65ea995f3ecac8:   done           |++++++++++++++++++++++++++++++++++++++| 
elapsed: 1.2 s                                                                    total:   0.0 B (0.0 B/s)        
```

### Rancher Desktop Version

1.4.1-main

### Rancher Desktop K8s Version

any

### Which container runtime are you using?

containerd (nerdctl)

### What operating system are you using?

Other Linux

### Operating System / Build Version

opensuse 15.3, TW

### What CPU architecture are you using?

x64

**Comment by jandubois** [comment]
I don't understand how this can happen; `nerdctl` runs inside the VM in the Alpine image, so it should not matter what OS you are running on the host; it only executes the wrapper script.

The only (somewhat far-fetched) idea I have is that you have `containerd` and `nerdctl` installed on your openSUSE system, and are running the system `nerdctl` instead of the wrapper.

Maybe check:

```console
$ file $(which nerdctl)
/Users/jan/.rd/bin/nerdctl: Bourne-Again shell script text executable, ASCII text
```

**Comment by mook-as** [comment]
It probably happens because we run curl internally:
https://github.com/rancher-sandbox/rancher-desktop/blob/afb05071d5cc17fb886f2926e9ca43e72947948b/src/assets/scripts/docker-credential-rancher-desktop#L11

We used to swallow stderr, but no longer do (for tests, I think):
https://github.com/rancher-sandbox/rancher-desktop/commit/f971683f6a370d55018e9009c8ffd19a528d6b4b#diff-28a12cd290fe71cc0c897ce159c05b84d0bba6c4d1eebddc416274faf313ae90

**Comment by jandubois** [comment]
I still don't understand why it depends on the host OS though. I've tried to repro by logging in and out of docker hub, but I never see this (on macOS).

**Comment by jandubois** [comment]
Also doesn't explain why it works with `rdctl shell` on openSUSE; this is basically the same thing the `nerdctl` bash wrapper does:

```
limactl shell 0 sudo --preserve-env=CONTAINERD_ADDRESS nerdctl "$@"
```

**Comment by jandubois** [comment]
Fixed by #2594
